### PR TITLE
BD-2500: Include user permissions for email self-service

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Comment by lydia-xie** [comment]
@cla-bot check

File: _docs/_user_guide/message_building_by_channel/email/email_setup/setting_up_ips_and_domains.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ After receiving your email, we'll get to work configuring your IPs, domains and
 
 ### Step 3: Add DNS records
 
-After your IPs, domains, subdomains, and IP pools are configured, we'll send you a list of DNS records. Ask your engineers and developers to add these DNS records where needed, and, after they have been added, let us know!
+After your IPs, domains, subdomains, and IP pools are configured, we'll send you a list of DNS records. Ask your engineers and developers to add these DNS records where needed, and, after they have been added, let the Braze Onboard team know.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/email/email_setup/setting_up_ips_and_domains.md:33]:
```suggestion
After your IPs, domains, subdomains, and IP pools are configured, we'll send you a list of DNS records. Ask your engineers and developers to add these DNS records where needed, and after they have been added, let the Braze Onboarding team know.
```

File: _docs/_user_guide/message_building_by_channel/email/email_setup/setting_up_ips_and_domains.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ This self-service email setup feature is currently in beta and only available fo
 
 Go to **Settings** > **Admin Settings** under **Company Settings**. Next, select the **Sender Verification** tab and click the **Start setup** button.
 
+Note to view the **Sender Verification** tab, you must have the "Can Manage Company Settings" company level permission.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/email/email_setup/setting_up_ips_and_domains.md:51]:
```suggestion
Note to view the **Sender Verification** tab, you must have the "Can Manage Company Settings" company-level permission.
```

**Review by rachel-feinberg — APPROVED** [review]
Left a small suggestion, but lgtm!

**Pull request merged**